test/repository/credential_repository_test.dart

```diff
@@ -0,0 +1,40 @@
+import 'dart:async';
+
+import 'package:flutter_templates/model/response/user_response.dart';
+import 'package:flutter_templates/repository/credential_repository.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+import '../mocks/generate_mocks.mocks.dart';
+
+void main() {
+  group('CredentialRepository', () {
```

Comment:
should this group name is `Test getProfile`? Then we will have another group is `Test updateProfile` for example 🤓

Reply:
@luongvo so for each function, we will test it in a group 💭 if so, I think we can go with `CredentialRepository - getProfile`. It's clearer and we are doing test so we don't need `Test` word, how do you think?

Reply:
@manh-t we don't need to repeat `CredentialRepository` as we're on this CredentialRepositoryTest already. Does that `CredentialRepository` prefix in the group name help on anything? 🤔

Reply:
@luongvo for example, when we run command `flutter test` and there is a failed test case. It's hard to see which test part is failed, unless we define them in the group name
![image](https://user-images.githubusercontent.com/60863885/133595544-e04bc2c4-13f7-4e29-812a-9102a990982c.png)

Reply:
@manh-t so it's okay to keep `CredentialRepository` or group them as `CredentialRepository - getProfile`